Alerting: Add feature toggles for Alertmanager migration (#77372)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `managedPluginsInstall`                     | Install managed plugins directly from plugins catalog                                                                                                                                                                                                                             |
| `prometheusPromQAIL`                        | Prometheus and AI/ML to assist users in creating a query                                                                                                                                                                                                                          |
| `alertmanagerRemoteSecondary`               | Enable Grafana to sync configuration and state with a remote Alertmanager.                                                                                                                                                                                                        |

> | `alertmanagerRemotePrimary`                 | Enable a remote Alertmanager instance as the primary Alertmanager.                                                                                                                                                                                                                |

Reviewer: ```suggestion
| `alertmanagerRemotePrimary`                 | Enable Grafana to have a remote Alertmanager instance as the primary Alertmanager.                                                                                                                                                                                                                |
```

> | `alertmanagerRemoteOnly`                    | Disable the internal Alertmanager and use an external one instead.                                                                                                                                                                                                                |

Reviewer: ```suggestion
| `alertmanagerRemoteOnly`                    | Disable the internal Alertmanager and only use the external one defined.                                                                                                                                                                                                                |
```
